src/modules/ChannelList/context/hooks/useFetchChannelList.ts

```diff
@@ -0,0 +1,71 @@
+import { useCallback } from 'react';
+import { GroupChannel, GroupChannelListQuery } from '@sendbird/chat/groupChannel';
+
+import { Nullable } from '../../../../types';
+import { CustomUseReducerDispatcher, Logger } from '../../../../lib/SendbirdState';
+import { MarkAsDeliveredSchedulerType } from '../../../../lib/hooks/useMarkAsDeliveredScheduler';
+import * as channelListActions from '../../dux/actionTypes';
+import { DELIVERY_RECEIPT } from '../../../../utils/consts';
+
+interface DynamicProps {
+  channelSource: Nullable<GroupChannelListQuery>;
+  premiumFeatureList: string[];
+  disableMarkAsDelivered: boolean;
+}
+interface StaticProps {
+  channelListDispatcher: CustomUseReducerDispatcher;
+  logSubtitle: string;
+  logger: Logger;
+  markAsDeliveredScheduler: MarkAsDeliveredSchedulerType;
+}
```

Comment:
dont we need to export DynamicProps and StaticProps? also name is not specific

Reply:
am... the props are not re-used in the other places, so I think we don't need to export for now